### PR TITLE
update Makefile to have rpm switch directories to rpmbuild dir before…

### PR DESCRIPTION
**Pull request by drumulonimbus**
… executing rpmbuild

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -292,6 +292,7 @@ rpm: install
 		$(CWD)/rexray.spec > $(RPMBUILD)/SPECS/rexray.spec
 
 	@printf "  ...building rpm..."; \
+		cd $)RPMBUILD); \
```
Comment (akutz) [inline Makefile]: Hi @drumulonimbus,

The line should be:

```
cd $(RPMBUILD); \
```

**Comment by akutz** [comment]
Hi @drumulonimbus,

I'm just circling back on this per our original conversation. I'm happy to approve this once you take the steps I outlined with regards to the commit message and multiple commits. Thanks!

**Comment by drumulonimbus** [comment]
Huh? I read all of the suggested links/pages and followed your instructions to a T yesterday, and resubmitted - let me retrace my steps and see where I've gone wrong, I'm guessing maybe I submitted the PR to my own fork or something.

**Comment by akutz** [comment]
You likely didn't force push, which would be required when squashing commits. So the last step likely failed.

**Pull request closed without merging**